src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -49,7 +51,29 @@
 Get_Kubernetes_Namespace_Fault_Type = 'kubernetes-get-namespace-error'
 Update_Agent_Success = 'Agents for Connected Cluster {} have been updated successfully'
 Update_Agent_Failure = 'Error while updating agents. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
-Cluster_Info_Not_Found_Type = 'Error while finding current cluster server details'
+Get_Credentials_Failed_Fault_Type = 'failed-to-get-list-cluster-user-credentials'
+Failed_To_Merge_Credentials_Fault_Type = "failed-to-merge-credentials"
+Kubeconfig_Failed_To_Load_Fault_Type = "failed-to-load-kubeconfig-file"
+Failed_To_Load_K8s_Configuration_Fault_Type = "failed-to-load-kubernetes-configuration"
+Failed_To_Merge_Kubeconfig_File = "failed-to-merge-kubeconfig-file"
+Different_Object_With_Same_Name_Fault_Type = "Kubeconfig has an object with same name"
+Download_Exe_Fault_Type = "Error while downloading client proxy executable from storage account"
+Create_Directory_Fault_Type = "Error while creating directory for placing the executable"
+Run_Clientproxy_Fault_Type = "Error while starting client proxy process."
+Post_Hybridconn_Fault_Type = "Error while posting hybrid connection details to proxy process"
+Post_RefreshToken_Fault_Type = "Error while posting refresh token details to proxy process"
+Merge_Kubeconfig_Fault_Type = "Error while merging kubeconfig."
+Create_CSPExe_Fault_Type = "Error while creating csp executable"
+Remove_Config_Fault_Type = "Error while removing old csp config"
+Load_Creds_Fault_Type = "Error while loading accessToken.json"
+Creds_NotFound_Fault_Type = "Credentials of user not found"
+Create_Config_Fault_Type = "Error while creating config file for proxy"
+Run_RefreshThread_Fault_Type = "Error while starting refresh thread"
+Load_Kubeconfig_Fault_Type = "Error while loading kubeconfig"
+Run_Check_CSP_Thread_Fault_Type = "Error while starting 'check csp thread'."
```

Comment:
>csp [](start = 63, length = 3)

Is this message getting printed for users? We dont expose terms like csp and all to them.

Reply:
@atharvamulmuley If required for customer issues debugging, can add it in logger.info(), this wont be visible to user directly. only when --debug is applied to the command, we would see these logs.

Reply:
we are using this fault type only in telemetry.set_exception() which wont expose it to end users. So we are good here.